content/docs/faq.md

```diff
@@ -71,17 +71,17 @@ Yes. Scratch Addons should not have any security issues in its most recent versi
 
 ### How can I report a security vulnerability?
 
-If you happen to find a security vulnerability please contact World_Languages privately by emailing `worldxlanguages (at) gmail.com`. If you don't get a response within 48 hours, please create an issue [here](https://github.com/ScratchAddons/ScratchAddons/issues/) mentioning you sent an email.
+If you happen to find a security vulnerability please contact World_Languages privately by emailing [`worldxlanguages@gmail.com`](mailto:worldxlanguages@gmail.com). If you don't get a response within 48 hours, please create an issue [here](https://github.com/ScratchAddons/ScratchAddons/issues/) mentioning you sent an email.
```

Comment:
The reason I did that is to avoid someone easily scraping the e-mail. (Not sure if that's still the case these days, but websites still hides and and ask for steps to get the e-mail.)

I this this should be reverted for now.

Reply:
why is scraping the email so important?

Reply:
Just like your phone number, so you don't get spammed with junk?

And having it with a link would be much easier for the bots to scrape it.

I think considering that I would try to improve it so it is more obfuscated, something on https://mattwthomas.com/blog/stop-email-scraping/

Reply:
Who would want to spam wl 🤨? Anyways, created a new PR #238

Reply:
> Who would want to spam wl 🤨?

Automated bots probably just crawl all websites and spam all emails.

Reply:
for fun?